src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -289,17 +476,79 @@
   word-break: break-word;
 }
 
-.pf-c-notification-drawer__group-toggle-count {
-  margin-right: var(--pf-c-notification-drawer__group-toggle-count--MarginRight);
-  margin-left: auto;
+// Task group toggle
+.pf-c-notification-drawer__task-group-toggle {
+  display: grid;
+  align-items: center;
+  width: 100%;
+  grid-template-columns: 1fr auto auto;
+  padding: var(--pf-c-notification-drawer__task-group-toggle--PaddingTop) var(--pf-c-notification-drawer__task-group-toggle--PaddingRight) var(--pf-c-notification-drawer__task-group-toggle--PaddingBottom) var(--pf-c-notification-drawer__task-group-toggle--PaddingLeft);
+  background-color: var(--pf-c-notification-drawer__task-group-toggle--BackgroundColor);
+  border: solid var(--pf-c-notification-drawer__task-group-toggle--BorderColor);
+  border-width: 0 0 var(--pf-c-notification-drawer__task-group-toggle--BorderBottomWidth) 0;
+  outline-offset: var(--pf-c-notification-drawer__task-group-toggle--OutlineOffset);
+}
+
+.pf-c-notification-drawer__task-group-header {
+  grid-column: 1 / 2;
+  grid-row: 1 / 2;
 }
 
-.pf-c-notification-drawer__group-toggle-icon {
-  margin-right: var(--pf-c-notification-drawer__group-toggle-icon--MarginRight);
-  color: var(--pf-c-notification-drawer__group-toggle-icon--Color);
-  transition: var(--pf-c-notification-drawer__group-toggle-icon--Transition);
```

Comment:
Removing these 3 vars will be a breaking change. If you wanted to share the vars, which seems safe here, you would want to use these existing ones.